lib/recurly/coupon.php

```diff
@@ -56,6 +68,17 @@ public static function deleteCoupon($couponCode, $client = null) {
     return Recurly_Base::_delete(Recurly_Coupon::uriForCoupon($couponCode), $client);
   }
 
+  protected function createUpdateXML() {
```

Comment:
I think what's here would work and it's probably what we'll end up going with but I'm going to poke at it a bit and see if I can think of anything cleaner. 

One thought was changing the `Recurly_Resource::xml()` function on to accept a method and maybe action so the coupon could just override that and if it was a PUT we'd do different XML.